lib/content/read.js

```diff
@@ -52,34 +54,33 @@ function readStream (cache, integrity, opts) {
 module.exports.hasContent = hasContent
 function hasContent (cache, integrity) {
```

Comment:
https://github.com/zkat/cacache/blob/latest/test/content.read.js#L135-L148 will need to be updated